src/test/java/NormalizerTest.java

```diff
@@ -0,0 +1,27 @@
+import filter.normalizer.*;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class NormalizerTest {
+    @Test
+    public void UpperCaseNormalizer(){
+        Normalizer upperCase=new UpperCaseNormalizer();
+        String str1="salam $293-AJS& bye das";
+        String str2="s]qw!` xxJSU USB usb _l";
+        String str3="=======2a";
+        Assertions.assertEquals(str1.toUpperCase(),upperCase.normalize(str1));
+        Assertions.assertEquals(str2.toUpperCase(),upperCase.normalize(str2));
+        Assertions.assertEquals(str3.toUpperCase(),upperCase.normalize(str3));
```

Comment:
try rewriting the uppercased version of String for expected value instead of using another method for doing it